vms/platformvm/block/executor/acceptor.go

```diff
@@ -110,7 +110,7 @@ func (a *acceptor) ApricotAtomicBlock(b *block.ApricotAtomicBlock) error {
 		zap.Stringer("blkID", blkID),
 		zap.Uint64("height", b.Height()),
 		zap.Stringer("parentID", b.Parent()),
-		zap.Stringer("utxoChecksum", a.state.Checksum()),
```

Comment:
The files in `platformvm` feel like an unrelated change... so I can take them out if requested. The rationale for coupling them in this PR was to keep the logging consistent.